client/src/components/Login.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { authAxios } from '../util/authAxios';
+import axios from 'axios';
+
+const Login = props => {
+  const [user, setuser] = useState({
+    username: '',
+    password: ''
+  });
+
+  const handleChange = e => {
+    setuser({ ...user, [e.target.name]: e.target.value });
+  };
+
+  const login = e => {
+    e.preventDefault();
+    axios
+      .post(`http://localhost:3030/auth/login`, user)
+      .then(res => {
+        localStorage.setItem('token', res.data.token);
+        props.history.push('/users');
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  };
+
+  return (
+    <>
+      <br />
+      <br />
+      <form>
+        <label>
+          Username:
+          <input
+            type='text'
+            name='username'
+            value={user.username}
+            onChange={handleChange}
+          />
+        </label>
+        <br />
+        <br />
+        <label>
+          Password:
+          <input
+            type='password'
+            name='password'
+            value={user.password}
+            onChange={handleChange}
+          />
+        </label>
+      </form>
+      <br />
+      <button onClick={login}>Login</button>
```

Comment:
Put the button inside of the `<form />` so that you can set login as the onSubmit property of form and set the button type to submit